presto-hive/src/main/java/io/prestosql/plugin/hive/util/HiveUtil.java

```diff
@@ -1056,4 +1060,14 @@ private static int getPositiveIntegerValue(Properties schema, String key, String
     {
         return toHiveTypes(schema.getProperty(IOConstants.COLUMNS_TYPES, ""));
     }
+
+    public static OrcWriterOptions getOrcWriterOptions(Properties schema, OrcWriterOptions orcWriterOptions)
+    {
+        if (schema.contains(ORC_BLOOM_FILTER_COLUMNS)) {
+            return orcWriterOptions
+                    .withBloomFilterColumns(ImmutableSet.copyOf(COLUMN_NAMES_SPLITTER.splitToList(schema.getProperty(ORC_BLOOM_FILTER_COLUMNS))))
+                    .withBloomFilterFpp(Double.parseDouble(schema.getProperty(ORC_BLOOM_FILTER_FPP)));
```

Comment:
I'd add error handling for this.  The property might not be present and might not be a double.  In either case, I'd just throw a `PrestoException` with `HIVE_UNSUPPORTED_FORMAT` error code.